Tools/Scripts/webkitpy/__init__.py

```diff
@@ -54,7 +54,7 @@
     AutoInstall.register(Package('importlib_metadata', Version(1, 7, 0)))
 
 AutoInstall.register(Package('atomicwrites', Version(1, 1, 5)))
-AutoInstall.register(Package('attr', Version(18, 1, 0), pypi_name='attrs'))
+AutoInstall.register(Package('attr', Version(20, 3, 0), pypi_name='attrs'))
```

Comment:
We'll need to make sure to run `test-webkitpy` on our Slackbot after landing this.